CHANGELOG.md

```diff
@@ -1,43 +0,0 @@
-# Changelog for new releases
```

Comment:
I think we must not delete the old changelog. The changes were not always in the release notes AFAIK.

Reply:
I checked and all the information is also contained in the rst